downstream/modules/eda/con-event-streams-attachment-activations.adoc

```diff
@@ -0,0 +1,39 @@
+
+[id="eda-event-streams-attachment"]
+
+= Attaching an event stream to a rulebook activation
+
+[role="_abstract"]
+Event streams can send events from remote systems to {EDAcontroller}. In a typical set-up, a server sends data to an event stream over the internet to an {EDAName} event stream receiver. When the data comes over the internet, the request must be authenticated. Depending on the webhook vendor or remote system, the authentication method could differ.
```

Comment:
the authentication method can differ.

Reply:
This file should have been removed. Sorry! The content was combined elsewhere. Deleting the file.